crates/sui-framework/packages/sui-framework/sources/bridge/bridge_committee.move

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#[allow(unused_use)]
+module sui::bridge_committee {
+    use std::vector;
+
+    use sui::ecdsa_r1;
+    use sui::vec_map::{Self, VecMap};
+
+    friend sui::bridge;
+
+    const ESignatureBelowThreshold: u64 = 0;
+
+    struct BridgeCommittee has store {
+        // commitee pub key and weight
+        pub_keys: VecMap<vector<u8>, u64>,
+        threshold: u64
+    }
+
+    public(friend) fun create_genesis_static_committee(): BridgeCommittee{
+        BridgeCommittee{
+            pub_keys: vec_map::empty<vector<u8>, u64>(),
+            threshold: 0
+        }
+    }
+
+    public fun verify_signatures(
+        committee: &BridgeCommittee,
+        message: vector<u8>,
+        signatures: vector<vector<u8>>
+    ) {
+        let (i, signature_counts) = (0, vector::length(&signatures));
+        let threshold = 0;
+        while (i < signature_counts) {
+            let signature = vector::borrow(&signatures, i);
+            let pubkey = ecdsa_r1::secp256r1_ecrecover(signature, &message, 1);
```

Comment:
what's the consideration to use `sha256` hash function here?

Reply:
Are there any preference for hash function? cc @benr-ml , @georgedigkas ?

Reply:
KECCAK256 is more secure, we should be using that, will change it.

Reply:
both are fine, where do you need it? i think that KECCAK256 is faster on eth if that's a requirement

Reply:
do you mean sign on `keccak(m)` vs `m`, but both using  `verify_with_hash::<Keccak256>`? I don't think there is much difference except for the extra hash, probably better to stick to one option everywhere